src/clients/node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tigerbeetle-node",
-  "version": "0.11.10",
+  "version": "0.11.11",
```

Comment:
This should be bumped in package-lock.json as well. You can run `npm i` and it will do it for you. We don't have a check for this cc @cb22 but we should. I don't know if there's an issue if they're not in step. But it seems pretty wrong to let them (though I've done this too).

Reply:
Thanks. :+1: